app/views/dashboard/_new_hosts_widget.html.erb

```diff
@@ -0,0 +1,11 @@
+<% hosts = Host.authorized(:view_hosts).includes(:owner, :operatingsystem).order(:created_at => :desc).limit(8) %>
```

Comment:
the dashboard data service already loads hosts respecting authorization and filters, you can use `@data.hosts.includes...`  instead and get the widget to respect the filter.